src/Razor/src/Microsoft.VisualStudio.Editor.Razor/RazorSyntaxTreePartialParser.cs

```diff
@@ -40,6 +40,20 @@ public PartialParseResultInternal Parse(SourceChange change)
             return result;
         }
 
+        /// <summary>
+        /// Gets the <see cref="RazorSyntaxTree"/> that the partial parser is operating on.
+        /// </summary>
+        /// <returns>The active <see cref="RazorSyntaxTree"/></returns>
+        /// <remarks>
+        /// The active <see cref="RazorSyntaxTree"/> has the <see cref="OriginalSyntaxTree"/>'s <see cref="RazorSyntaxTree.Source"/> and <see cref="RazorSyntaxTree.Diagnostics"/>.
+        /// </remarks>
+        public RazorSyntaxTree GetActiveSyntaxTree()
+        {
+            var activeSyntaxTree = RazorSyntaxTree.Create(ModifiedSyntaxTreeRoot, OriginalSyntaxTree.Source, OriginalSyntaxTree.Diagnostics, OriginalSyntaxTree.Options);
```

Comment:
From offline convo with @rynowak, just return this always as part of the partial parse result.